mobile-app/cypress/support/loanCommands.ts

```diff
@@ -64,44 +42,63 @@ export function checkCollateralCardValues(
   dollarValue: string,
   vaultShare: string
 ): void {
-  cy.getByTestID(`collateral_card_symbol_${symbol}`).contains(symbol);
-  cy.getByTestID(`collateral_card_col_amount_${symbol}`).contains(amount);
-  cy.getByTestID(`collateral_card_col_amount_usd_${symbol}`).contains(
-    dollarValue
+  cy.getByTestID(`vault_detail_collateral_${symbol}_amount`).contains(amount);
+  cy.getByTestID(`vault_detail_collateral_${symbol}_usd`).contains(dollarValue);
+  cy.getByTestID(`vault_detail_collateral_${symbol}_vault_share`).contains(
+    vaultShare
   );
-  cy.getByTestID(`collateral_card_vault_share_${symbol}`).contains(vaultShare);
 }
 
 export function checkVaultDetailValues(
-  status: string,
   vaultID: string,
   totalCollateral: string,
+  maxLoanAmount: string,
   totalLoans: string,
-  vaultInterest: string
+  vaultInterest: string,
+  minColRatio: string,
+  status?: string,
+  vaultRatio?: string
 ): void {
-  cy.getByTestID(`vault_detail_status`).contains(status);
-  cy.getByTestID(`vault_detail_id`).contains(vaultID);
-  cy.getByTestID(`text_total_collateral_value`).contains(totalCollateral);
-  cy.getByTestID(`text_total_loan_value`).contains(totalLoans);
-  cy.getByTestID(`text_vault_interest`).contains(vaultInterest);
+  if (status !== undefined) {
+    cy.getByTestID("vault_status").contains(status);
+  }
+  if (vaultRatio !== undefined) {
+    cy.getByTestID("vault_ratio").contains(vaultRatio);
+  }
+  cy.getByTestID("collateral_vault_id").contains(vaultID);
+  cy.getByTestID("total_collateral").contains(totalCollateral);
+  cy.getByTestID("max_loan_amount").contains(maxLoanAmount);
+  cy.getByTestID("total_loan").contains(totalLoans);
+  cy.getByTestID("interest").contains(vaultInterest);
+  cy.getByTestID("min_col_ratio").contains(minColRatio);
 }
 
 export function checkVaultDetailCollateralAmounts(
   amount: string,
   displaySymbol: string,
   vaultShare: string
 ): void {
-  cy.getByTestID(`vault_detail_collateral_${displaySymbol}`).contains(
-    displaySymbol
-  );
+  // cy.getByTestID(`vault_detail_collateral_${displaySymbol}`).contains(
```

Comment:
what's this for?

Reply:
outdated checking since we are not showing the display symbol, in collateral cards, will update this

Reply:
removed commented code, and updated duplicated function for checking collaterals in vault details